src/egon/data/processing/wind_farms.py

```diff
@@ -0,0 +1,477 @@
+from egon.data import db
+import geopandas as gpd
+import pandas as pd
+import numpy as np
+from matplotlib import pyplot as plt
+from shapely.geometry import Polygon, LineString, Point, MultiPoint
+
+
+def wind_power_parks():
+    """ Main function. Import power objectives generate results calling the
+    functions "generate_wind_farms" and  "wind_power_states".
+    
+    Parameters
+    ----------
+    *No parameters required
+    
+    """
+    
+    con = db.engine()
+
+    # federal_std has the shapes of the German states
+    sql = "SELECT  gen, gf, nuts, geometry FROM boundaries.vg250_lan"
+    federal_std = gpd.GeoDataFrame.from_postgis(sql, con, geom_col="geometry", crs=4326)
+
+    # target_power_df has the expected capacity of each federal state
+    sql = "SELECT  carrier, capacity, nuts, scenario_name FROM supply.egon_scenario_capacities"
+    target_power_df = pd.read_sql(sql, con)
+
+    # mv_districts has geographic info of medium voltage districts in Germany
+    sql = "SELECT geom FROM grid.mv_grid_districts"
+    mv_districts = gpd.GeoDataFrame.from_postgis(sql, con)
+
+    # Delete all the water bodies from the federal states shapes
+    federal_std = federal_std[federal_std["gf"] != 1]
```

Comment:
@CarlosEpia @KathiEsterl @ClaraBuettner if you use GF 2,3 and 4, you still have some water bodies in the data, e.g. Flensburger Förde (cf. [VG250 docs](https://sg.geodatenzentrum.de/web_download/vg/vg250-ew_3112/vg250-ew_3112.pdf)).
As the RE potential data should not contain data in those areas, your results are most likely not affected but to be sure, use GF=4 to use land only..